tests/app/dao/test_jobs_dao.py

```diff
@@ -192,23 +193,26 @@ def test_get_jobs_for_service_with_limit_days_edge_case(notify_db, notify_db_ses
     assert job_eight_days_old not in jobs_limit_days
 
 
-def test_get_jobs_for_service_in_created_at_order(notify_db, notify_db_session, sample_template):
-    job_1 = create_job(
-        notify_db, notify_db_session, sample_template.service, sample_template, created_at=datetime.utcnow())
-    job_2 = create_job(
-        notify_db, notify_db_session, sample_template.service, sample_template, created_at=datetime.utcnow())
-    job_3 = create_job(
-        notify_db, notify_db_session, sample_template.service, sample_template, created_at=datetime.utcnow())
-    job_4 = create_job(
-        notify_db, notify_db_session, sample_template.service, sample_template, created_at=datetime.utcnow())
+def test_get_jobs_for_service_in_processed_at_then_created_at_order(notify_db, notify_db_session, sample_template):
+    _create_job = partial(create_job, notify_db, notify_db_session, sample_template.service, sample_template)
+    scheduled_jobs, processed_jobs = [], []
+
+    for index in range(0, 4):
+        scheduled_jobs.append(_create_job(
+            created_at=datetime.utcnow() - timedelta(seconds=index)
+        ))
+        processed_jobs.append(_create_job(
+            created_at=datetime.utcnow() - timedelta(minutes=(4 - index)),
+            processing_started=datetime.utcnow() - timedelta(hours=index)
+        ))
```

Comment:
not a huge fan of this test - it's pretty hard to read and work out what the actual times involved are - can we unroll this? (we also shouldn't need 8 entries to test a sort)